### DISCUSSION TITLE
monitor screen dark without shader or texture pack

### DISCUSSION THREAD

**Discussion by demonchild42** [cc-tweaked/CC-Tweaked#1720] [Q&A]
i have been playing the pack create:stellar on a server and the monitors are all really dark i have found things online saying it could be caused by shaders or texture packs but i have all of them either disabled or deleted and i still can barely see any text. are there any other things that could cause it?

**Comment by demonchild42** [comment]
![Screenshot 2024-02-10 193913](https://github.com/cc-tweaked/CC-Tweaked/assets/54030972/8c31d893-39c8-477f-9b1a-fd9600101a35)
this is with base red text and a base white background

**Reply by Wojbie** [reply]
This looks like you have a shaders mod installed and no shaders activated. In case of some shaders mod if there is no active shader they will cause CC:T monitors to be way darker then they should be. You should try removing shaders mod or finding and activating a shader that don't break monitors.
